concourse/client/util.py

```diff
@@ -71,14 +72,22 @@ def has_job_been_triggered(
     job: Job,
     resource_version: ResourceVersion,
     concourse_api,
+    retry: int=6,
+    sleep_sec: int=5,
```

Comment:
please do not abbreviate the unit (seconds); this is inconsistent with our codebase, and also I do not think the benefit of saving a few keystrokes is worth the potentially worse readability

in other places in our codebase, such a symbol would be named `sleep_time_seconds`